### PR TITLE
ADEN-7139 Updates Prebid integration to v1.11

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

**Review by dmnsobczak — APPROVED** [review]

**Comment by leshniak** [comment]
LGTM

File: extensions/wikia/AdEngine/js/lookup/prebid/prebid.js [hunk 1]
```diff
@@ -42,10 +42,6 @@ define('ext.wikia.adEngine.lookup.prebid', [
 		biddersPerformanceMap = performanceTracker.setupPerformanceMap(skin);
 		adUnits = helper.setupAdUnits(skin, isLazyLoadingEnabled ? 'pre' : 'off');
 
-		if (win.pbjs) {
-			win.pbjs._bidsReceived = [];
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/prebid.js]: What happens on mobile-wiki on consecutive pvs?

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/prebid.js]: We checked we can see that after page change Prebid is removing old bids and calling for new one. So this scenario is fine or was "accidentally" fine before without this chunk of code.

File: extensions/wikia/AdEngine/js/lookup/prebid/prebid.js [hunk 2]
```diff
@@ -34,6 +36,16 @@ define('ext.wikia.adEngine.lookup.prebid', [
...
 
 	function call(skin, onResponse) {
+		if (!isPrebidVersionChoosed) {
+			if (isNewPrebidEnabled) {
+				loadNewPrebid();
+			} else {
+				loadOldPrebid();
+			}
+
+			isPrebidVersionChoosed = true;
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/prebid.js]: Choosen :)

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/prebid.js]: :D 
Fixed.

**Review by fraszczakszymon — APPROVED** [review]

**Review by dmnsobczak — APPROVED** [review]

**Comment by leshniak** [comment]
LGTM

**Pull request merged**